core/templates/dev/head/components/summary_tile/story_summary_tile_directive.html

```diff
@@ -0,0 +1,44 @@
+<a ng-href="<[getStoryLink()]>">
+  <div class="oppia-story-image">
+    <img src="https://image.ibb.co/iUeEUJ/Screenshot_from_2018_07_21_11_07_04.png" alt="">
```

Comment:
Don't link to an external site. Put an image in assets/ and use a path to it for now. It's better for it to be a dummy image, like a question mark or something, so that it serves as a reminder for us to fix it before launching. Also please file an issue to change it to a creator-uploaded image instead.

Reply:
Done